test-cases/gemini/gemini-basic-3h.yaml

```diff
@@ -1,20 +1,20 @@
 test_duration: 500
 n_db_nodes: 1
 n_loaders: 1
-n_monitor_nodes: 1
+n_monitor_nodes: 0
 instance_type_db: 'i3.large'
 
 user_prefix: 'gemini-basic-3h'
-failure_post_behavior: keep
+failure_post_behavior: destroy
 store_results_in_elasticsearch: False
 # gemini
 # cmd: gemini -d -n [NUM_OF_TEST_ITERATIONS] -c [NUM_OF_THREADS] -p [NUM_OF_PARTITION_KEYS_PER_THREAD] -m mixed -f
 # the below cmd runs about 3 hours
-gemini_cmd: "gemini -d -n 500000 -c 20 -p 100 -m mixed -f"
-gemini_url: 'https://s3.amazonaws.com/scylla-gemini/Binaries/gemini-latest'
+gemini_cmd: "gemini -d --duration 10800s -c 20 -p 100 -m mixed -f"
+gemini_url: 'http://downloads.scylladb.com.s3.amazonaws.com/gemini/0.9.2/gemini_0.9.2_Linux_x86_64.tar.gz'
 gemini_static_url: 'https://s3.amazonaws.com/scylla-gemini/Binaries/schema.json'
 
 db_type: mixed_scylla
-instance_type_db_oracle: 'i3.8xlarge'
-ami_id_db_oracle: 'ami-0081311880c437b65' # ScyllaDB 2.3.3 us-east-1 AMI
+instance_type_db_oracle: 'i3.large'
```

Comment:
Why did you change this for the test_oracle?

Reply:
After talk with Pekka, he said that there is not different requirement to test_db_oracle instance. So i decided to have same instances for better test result correlation.

Reply:
There is a difference with how we run it.
We want that all the data to fit in memory so the the test_oracle won't involve disk operations.

Reply:
return i3.8xlarge